### DISCUSSION TITLE
Scrap Heap

### DISCUSSION THREAD

**Discussion by david-roxygen** [secretcowlevel/DoomtrooperPublic#78] [Card Balance]
Current version:
Scrap Heap (CYBERTRONIC)
SUPPORT
When a cybertronic warrior is killed, gain 1 scrap counter.
1 action + 1 counter: augment target cybertronic warrior.

-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-
Suggested version:
Scrap Heap (CYBERTRONIC)
SUPPORT
Annihlate your discard pile. For every 5 cards Annihlated, gain 1 scrap counter.
1 action +1 scrap counter: augment target cybertronic warrior.

-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-

**Comment by 21-ric** [comment]
What do you think if we use Scrap Heap during the post combat phase? 🤠

Scrap Heap (CYBERTRONIC) [R] [#155]
INFLUENCE
When a Cybertronic Warrior is killed in combat, target another Cybertronic WARRIOR : AUGMENT 1.

**Reply by david-roxygen** [reply]
That is similar to the current version with 0 action to give an augment to a target cybertronic warrior.

The part of this card that is tricky for me is the link with a killed warrior.  If your opponent kill more than 1 of your warriors, probably you are near to lose a match. It is difficult that this card could make the difference.

**Reply by 21-ric** [reply]
We would have already found a solution if this card wasn't tricky.

**Comment by david-roxygen** [comment]
Changing a bit the idea behind this card,  it could be converted in a fate card able to assign an augment to target  cybertronic warrior. Something like:
-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-
Suggested version 2:
Scrap Heap (CYBERTRONIC)
FATE
1 Action: augment target cybertronic warrior.
-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-
A simple and useful effect. Could be possible use it in combo with ‘at your service’ and ‘mirrorman’.
On mirrorman it has a double effect
I proposed the cost of 1 action to restrict this card out of the combat phase (even if a version with 0 action cost maybe could be acceptable, I don’t know…)

**Comment by Sol-au-fein** [comment]
I think it can be solved if it comes to play with one counter already on it. Recharging by killed warrior remains. Idea behind this card indicates it has to be support.

**Comment by Sol-au-fein** [comment]
Scrap Heap
1 Action: Discard CHASSEUR from hand to Augment target CHASSEUR once.

**Comment by david-roxygen** [comment]
The idea of discard CHASSEUR from the hand to assign augments is nice.
Thinking about it, probably the most useful version could be a card that start with counters on it.
Based on previous suggestion:

-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-
Current version:
Scrap Heap (CYBERTRONIC)
SUPPORT
INFLUENCE
When a cybertronic warrior is killed, gain 1 scrap counter.
1 action + 1 counter: augment target cybertronic warrior.

-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-
Suggested version 3 (roxygen):
Scrap Heap (CYBERTRONIC)
SUPPORT
INFLUENCE RESTRICTED
Start with 3 counters. 1 action + 1 counter: augment target CHASSEUR warrior.
When a CHASSEUR warrior is killed, gain 1 scrap counter.

The focus on CHASSEUR could be useful to improve the general strategy with an hypothetical chasseur deck.
